index.html

```diff
@@ -24,14 +26,130 @@ <h1>I’m Mariane
+      <div></div>
+      <p >Back End Dev</p>
+      <div></div>
+      <p>2015</p>
+     </div>
+      <p class="text">A daily selection 
+       of privately personalized reads; no accounts or sign-ups required.</p>
+     <ul class="lang">
+      <li>html</li>
+      <li>css</li>
+      <li>javaScript</li>
+     </ul>
+     <button type="button">See Project</button>
+    </article>
+    <article>
+     <div class="snapshoot"><img src="images/Snapshoot3.png" alt=""></div>
+     <h3>Multi-Post Stories</h3>
+     <div class="frame">
+      <h4>CANOPY</h4>
+      <div></div>
+      <p >Back End Dev</p>
+      <div></div>
+      <p>2015</p>
+     </div>
+      <p class="text">A daily selection 
+       of privately personalized reads; no accounts or sign-ups required.</p>
+     <ul class="lang">
+      <li>html</li>
+      <li>css</li>
+      <li>javaScript</li>
+     </ul>
+     <button type="button">See Project</button>
+    </article>
+   </section>
+   <section class="Fabout">
+    <h2>About<br>Myself</h2>
+    <p>Hello I’m a software developer! 
+      I can help you build a product , 
+      feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+      don’t hestiate to contact me.</p>
+    <a href="">LET’S CONNECT</a>
+    <ul class="social_media">
+     <li><a href=""><i class="fab fa-github"></i></a></li>
+     <li><a href=""><i class="fab fa-linkedin"></i></a></li>
+     <li><a href=""><i class="fab fa-angellist"></i></a></li>
+     <li><a href=""><i class="fab fa-twitter"></i></a></li>
+     <li><a href=""><i class="fab fa-medium-m"></i></a></li>
+    </ul>
+    <button type="button">Get my resume</button>
+    <article class="container-lan">
+     <div class="lan-tle tags"><h3>Langguages</h3><i class="fa fa-angle-down"></i></div>
+     <div class="toggle">
+      <div class="lan"><img src="images/js.png" alt=""><p>JavaScript</p></div>
+      <div class="lan"><img src="images/html.png" alt=""><p>HTML</p></div>
+      <div class="lan"><img src="images/css.png" alt=""><p>CSS</p></div>
+     </div>
+     <div class="lan-tle"><h3>Frameworks</h3><i class="fa fa-angle-right"></i></div>
+     <div class="lan-tle"><h3>Skills</h3><i class="fa fa-angle-right"></i></div>
+    </article>
+   </section>
   </main>
-  <footer></footer>
+  <footer class="contact">
+   <div class="txt">
+    <h2>Contact me</h2>
+    <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+   </div>
+   <form action="index.html">
+    <input type="text" name="name" id="name" placeholder="Yeremias">
+    <input type="email" name="email" id="email" placeholder="natanjayanj14@gmail.com">
+    <textarea name="message" id="message" cols="30" rows="10">Write your message here</textarea>
+    <input type="submit" value="Get In Touch">
+   </form>
+   <div class="line"></div>
+  </footer>
  </body>
```

Comment:
[OPTIONAL]: Please ensure that your contact section should not be wrapped in `<footer>`. Kindly remove the contact section from `<footer>`.